data_structures/src/data_request.rs

```diff
@@ -391,7 +391,6 @@ pub fn calculate_tally_change(
 
 pub fn calculate_witness_reward(
     commits_count: usize,
-    reveals_count: usize,
```

Comment:
remember not remove it, to be able to use it with the after_second_hard_fork branch